lib/hash_digest.rb

```diff
@@ -0,0 +1,29 @@
+require 'digest'
+
+class HashDigest
```

Comment:
Minor, but I'd suggest putting this as a model concern or under the namespace of SubscriberList to associate it more with the domain it's within. Being in `lib` makes it seem generic but it's rather specific to SubscriberList.